bot/core/bot.py

```diff
@@ -1,20 +1,30 @@
 from datetime import datetime
 
+import asyncpg
 from discord import Color, Embed
 from discord.ext.commands import Bot as Base_Bot
 
 from bot import config
 
 
 class Bot(Base_Bot):
+    """Subclassed Hotwired bot."""
+
     def __init__(self, extensions: list, *args, **kwargs) -> None:
+        """Initialize the hotwired bot."""
```

Comment:
We don't require a comment on `__init__` functions and this comment is obviously only a place holder, it serves no real purpose (doesn't describe anything more that is clearly visible from the code)

Reply:
My linters require one, so I did put one

Reply:
You should add that requirement to the ignored errors/warnings for your linter since this docstring is very redundant and there is really no need to keep it